resources/main.mjs

```diff
@@ -290,6 +291,36 @@ class MainBenchmarkClient {
         csvLink.setAttribute("download", `${filePrefix}.csv`);
     }
 
+    _populateNonStandardParams() {
+        if (params === defaultParams)
+            return;
+        const paramsDiff = [];
+        const usedSearchparams = params.toSearchParamsObject();
+        const defaultSearchParams = defaultParams.toCompleteSearchParamsObject(false);
+        for (const [key, value] of usedSearchparams.entries()) {
+            if (key === "developerMode")
+                continue;
+            const defaultValue = defaultSearchParams.get(key);
+            if (value !== defaultValue)
+                paramsDiff.push({ key, value, defaultValue });
+        }
+        if (paramsDiff.length === 0)
+            return;
+        let body = "";
+        for (const { key, value, defaultValue } of paramsDiff)
+            body += `<tr><th>${key}</th><th>${value}</th><th>${defaultValue}</th></tr>`;
```

Comment:
Can we escape each value so that we won't have a XSS vulnerability?

Reply:
done.